src/text/matches.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Return matches against regexp
+ * @param {string} target
+ * @param {string} regExp
+ * @return Boolean
```

Comment:
The returned value is an array, not a boolean

Reply:
fixed